index.html

```diff
@@ -5,11 +5,9 @@
     <title>Fork Me! FCC: Test Suite Template</title>
     <link rel="stylesheet" href="css/style.css">
   </head>
-
-  <body>
-
-   
-    <h1 id="title">freeCodeCamp Survey Form</h1>
+<body>
+<section>
+   <h1 id="title">freeCodeCamp Survey Form</h1>
     <p id="description">Thank you for taking the time to help us improve the platform</p>
     <div class="container">
       <form id="survey-form">
```

Comment:
- [ ] <b>OPTIONAL:</b> This is an example as of how to use inline comments for optional change.